.github/workflows/pr-linter-and-tests.yml

```diff
@@ -35,6 +35,10 @@ jobs:
         with:
           ref: ${{ github.event.pull_request.head.sha }}
           fetch-depth: 2
+      - name: Allow access to python-gitlab fork
+        uses: webfactory/ssh-agent@v0.9.0
+        with:
+          ssh-private-key: ${{ secrets.SSH_PRIVATE_KEY }}
```

Comment:
Is this (and similar ones below) block really needed, @TimKnight-DWP? I don't see the usage of a python-gitlab fork below.

I am also afraid that it will break CI/CD in the main branch after it gets merged, as we don't have a SSH_PRIVATE_KEY secret set in this project.

Reply:
@gdubicki could've sword I'd reverted this out, is something that I needed in: https://github.com/TimKnight-DWP/gitlabform/pull/1, because my pyhton-gitlab changes are not released yet _but_ shouldn't be in here

Reply:
ohhhh yeah it's on my `main` branch of fork in order to test the above PR, so when I rebased that way it came back in, DOH